### PR TITLE
Refactor Dx

### PR DESCRIPTION
**Pull request by rernst**
- tabs after input/output/script
- file -> path
- Add emit to outputs

### CODE REVIEW

File: BWA/0.7.17/BWASW.nf [hunk 1]
```diff
@@ -6,13 +6,13 @@ process BWASW {
     shell = ['/bin/bash', '-euo', 'pipefail']
 
     input:
-    tuple sample_id, rg_id, file(fastq: "*")
```
Comment (sawibo) [inline BWA/0.7.17/BWASW.nf:9]: I believe path(fastq:"*") should be path(fastq) , I've tried fastq:("*") and it didn't work

File: BWA/0.7.17/MEM.nf [hunk 2]
```diff
@@ -6,16 +6,16 @@ process MEM {
     shell = ['/bin/bash', '-euo', 'pipefail']
 
     input:
-    tuple sample_id, rg_id, file(fastq: "*")
+        tuple(sample_id, rg_id, path(fastq: "*"))
```
Comment (sawibo) [inline BWA/0.7.17/MEM.nf:9]: See BWASW for comments

File: FastQC/0.11.8/FastQC.nf [hunk 3]
```diff
@@ -5,13 +5,13 @@ process FastQC {
     shell = ['/bin/bash', '-euo', 'pipefail']
 
     input:
-    tuple sample_id, rg_id, file(fastq: "*")
+        tuple(sample_id, rg_id, path(fastq: "*"))
```
Comment (sawibo) [inline FastQC/0.11.8/FastQC.nf:8]: path(fastq: "*") ->  path(fastq)

File: MultiQC/1.8/MultiQC.nf [hunk 4]
```diff
@@ -5,14 +5,14 @@ process MultiQC {
     shell = ['/bin/bash', '-euo', 'pipefail']
 
     input:
-    tuple val(analysis_id), file(qc_files: "*")
```
Comment (sawibo) [inline MultiQC/1.8/MultiQC.nf:8]: path(qc_files:"*") -> path(qc_files)

File: BWA/0.7.17/BWASW.nf (hunk 1 above)
Comment (rernst) [reply BWA/0.7.17/BWASW.nf:9]: Fixed :)

**Pull request merged**